### PR TITLE
add CONTRIBUTING, CoC, update License, Style Guide

### LINKED ISSUE #531: JOSE review feedback
Hi @abhijna :wave: 

I'm one the reviewers for your JOSE paper https://github.com/openjournals/jose-reviews/issues/167

I'm planning to do a full run through of the AWS course myself so will have more feedback later, but I've got a few initial comments.

## Use of admin credentials for participants
https://github.com/nih-cfde/training-and-engagement/wiki/IAM-account-for-AWS

It makes things easier for to setup, especially if you have additional AWS courses, but it does expose the AWS account holder to the risk of misuse. Best-practice is to minimise privileges where reasonable. Since the participants are potentially complete novices there's a higher risk of leaking credentials, and there's the risk of someone playing around and creating other unexpected resources.

Have you considered adding a restrictive IAM policy? For instance you could allow just console access, `ec2:*`, and whatever else is required, along with a region restriction (it's annoying easy to accidentally switch region without realising).

## How others should re-use your course materials
Do you recommend people use https://training.nih-cfde.org/en/latest/Cloud-Platforms/Introduction_to_Amazon_Web_Services/introtoaws1/ directly, or would you prefer them to build their own site using your materials? I noticed in an older version of the course you had a link to a Google feedback form (now removed), which obviously isn't relevant for other institutions, so I was wondering what your preferences were for re-use.

### PR DESCRIPTION
**Pull request by raynamharris**
## PR Checklist

- [x] Release label added
- [x] PR category label added
- [x] PR mergeable
- [x] Clean build logs
- [x] Linked related issues #531 #532 

## PR Description

This PRs adds a contributing file as per [#531 ](https://github.com/nih-cfde/training-and-engagement/issues/531#issuecomment-1155022385). 

Also added the dependent CoC and license files.

Also renamed the index.md file in the Website-Style-Guide directory so that the landing page for the website style guide is https://training.nih-cfde.org/en/latest/Website-Style-Guide/ instead of https://training.nih-cfde.org/en/latest/Website-Style-Guide/0index/

## PR Preview 

https://cfde-training-and-engagement--542.com.readthedocs.build/en/542/Website-Style-Guide/

## Reviewer Checklist 

- [x] spelling and grammar
- [x] functional links (inter and intra)

## Timeline

June release

### CODE REVIEW

**Comment by raynamharris** [comment]
started as an an update of the contributing file, ended with some updates to the rendered style guide
Screen Shot 2022-06-16 at 12 31 09 AM

**Comment by jeremywalter** [comment]
Website Style Guide doesn't show up for me on the menus or TOC. I can see it rendered when I use the link up top, but that's because it explicitly names the page, otherwise it's hidden.

**Comment by raynamharris** [comment]
working as intended. various documents link to it, such as the contributing and read me files and the wiki. but, we don't need it in the main menu bar.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ A list of a past workshops can be viewed [here](https://github.com/nih-cfde/trai
```
Comment (jessicalumian) [inline README.md:20]: line 19 of README.md should be "a list of past workshops" (get rid of a)

**Review by jeremywalter — APPROVED** [review]

**Pull request merged**